.dependabot/config.yml

```diff
@@ -0,0 +1,6 @@
+version: 1
+update_configs:
+  - package_manager: "dotnet:nuget"
+    directory: "/"
+    update_schedule: "weekly"
```

Comment:
This is a change from how we currently have things configured. Is this an intentional change?

Reply:
My bad. The configured value on the dependable app is daily. I'll change it.